group-coordinator/src/test/java/org/apache/kafka/coordinator/group/consumer/ConsumerGroupMemberTest.java

```diff
@@ -17,10 +17,11 @@
 package org.apache.kafka.coordinator.group.consumer;
```

Comment:
Should we add a unit test here to verify that non-existent partitions are not provided in the response?

Reply:
Yeah, we have that in `testAsConsumerGroupDescribeWithTopicNameNotFound`.